docs/faq/events.md

```diff
@@ -28,3 +30,182 @@ There are lots of times when words just aren't enough to really get clarity on a
     style="height: 100%; width: 100%; border: 0;">
   </iframe>
 </div>
+
+## Event listening
+
+When preparing to listen for events in your code, do the least work possible. Not only because this allows you to do less work now, but it allows you also to do less work later, and most importantly it allows the browser to do less work, which means a more performant application or component. The leasst work possible changes by context, but in important to remember no matter where you are adding event listners.
+
+### Inside of your elements
+
+When adding event listening inside of your components, assuming you are using `lit-html` (or any other declarative renderer with event binding syntax) use the tools supplied therein to [add event listeners](https://lit-html.polymer-project.org/guide/writing-templates#add-event-listeners). The reasoning behind this is that for any listener added by `lit-html`, the same listener will be removed by `lit-html` when no longer needed; no ghost listeners! That means whether the DOM and its associated lister is available for the entire lifecycle of your application, or just for a small part of that, the least amount of work will be done to ensure the event is handled appropriately. See the `click` listener in the following code. It _only_ exits while `this.open` is `true`, which leaves the browser listening to less things and your code less likely to trigger handlers errantly.
+
+```js
+render() {
+    return html`
+        ${this.open
+            ? html`<button @click=${this.handleClose}>Close</button>
+            : html``
+        }
+    `;
+}
+```
+
+A side benefit of relying on the `@event` syntax in `LitElement` (if you use it) is that all events handled therein are automatically bound to `this` which will prevent the need to do so manually: no `this.handleClose = this.handleClose.bind(this);` and, no `handleClose = (event) => {}` in the case above.
+
+### On your elements
+
+If you are adding a listener to your element, from inside your element, you should add that listener in the `constructor`. The browser does a great job of garbage collecting that listener when the element is removed from the DOM, so there no reason to do any work to manage that yourself. See the following example where the element listeners for a `done` event.
+
+```js
+constructor() {
+    super();
+    this.addEventListener('done', this.handleDone);
+}
+handleDone = (event) => {
```

Comment:
Prefer to use the bind syntax here. Public properties are not supported everywhere yet, people may come to this page, scan it for what they need, copy + paste code, and go "it doesnt work!"